issue_processing/issue_processing.py

```diff
@@ -79,13 +79,13 @@ def run():
     # 1) load the list of issues
     issues = load(__srcdir)
     # 2) re-format the issues
-    issues = reformat_issues(issues)
+    reformat_issues(issues)
```

Comment:
Can you please apply this change (and all the related ones below) also to the corresponding functions in jira_issue_processing?